MDCLogArchitecture/MDCLog.Models/DomainModels/Interfaces/Repositories/Interface1.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MDCLog.Models.DomainModels.Filters;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MDCLog.Models.DomainModels.Interfaces.Repositories
+{
+    interface ILogComments
```

Comment:
My only comment would be to make this singular instead of plural. so it represents just one instance of a comment